### DISCUSSION TITLE
Help getting this integration working

### DISCUSSION THREAD

**Discussion by martinoturrina** [gicamm/homeassistant-comelit#55] [Q&A]
Hi, looks like this integration is awesome but I'm too stupid to understand how to set it up properly.

I was able to install it via HACS, and to add the configuration in the configuration file but I'm failing to understand where to gather the needed information to insert in the configuration.

I've installed this comelit HUB https://pro.comelitgroup.com/it-it/prodotto/20003150 and when following the procedure to find the serial I can find it but is has a letter in it, is it possible?

Second question, where I can find username and password? Are these the credential for the app login?

Currently in the logs I see 

`2023-11-28 22:21:50.838 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: comelit
2023-11-28 22:21:50.977 ERROR (Thread-3 (_thread_main)) [custom_components.comelit.hub] Error dispatching {'req_type': 5, 'req_sub_type': -1, 'seq_id': 3, 'req_result': 4, 'message': 'LOGIN ERROR'}
2023-11-28 22:21:50.978 ERROR (Thread-3 (_thread_main)) [custom_components.comelit.hub] 'sessiontoken'
2023-11-28 22:21:52.525 ERROR (Thread#BS) [custom_components.comelit.vedo] Error getting data! None: Max retries exceeded with url: /login.cgi?_=1701206510928 (Caused by None)
2023-11-28 22:22:22.960 ERROR (Thread#BS) [custom_components.comelit.vedo] Error getting data! None: Max retries exceeded with url: /login.cgi?_=1701206542533 (Caused by None)`

So I think I'm stuck with logging in

Thanks for any help!

**Comment by gicamm** [comment]
Hi Martino,
It looks like you are using bad credentials.
```
comelit:
  hub:
    username: XXX
    password: XXX
```

The username and password are the same as those you use to log in to the application.

The serial is something like HSRV-xxxxxxx. You only need numbers.

**Comment by martinoturrina** [comment]
Hi thanks for the answer, 

since in the app I log in with email and psw of the comelit account, I tried that but it was strange to me that the online account credentials were needed. In a desperation move I tried admin admin as user and psw and it worked :)

Additionally my serial was showing like this (with a final F letter) HSRV - xx.xx.xx.xx.xx.xF and it worked by using xxxxxxxxxxxF so with the final letter included. Seems like it is only needed to remove the "HSRV -" part of it.

BTW integration looks great, didn't expect to be able to see the temperatures of my climate control system in HA but thanks to your work I did it ❤️

**Comment by gicamm** [comment]
Great :D
